### DISCUSSION TITLE
use "two packager" in one thread

### DISCUSSION THREAD

**Discussion by mehrdad2000** [tilln/jmeter-iso8583#95] [General]
Hi
I need to use two packager in one thread, in this thread I have two sampler that generate two different message that packager not same.

is it possible to add two ISO8583 Connection Configuration" how it work? how set which packager related to which sampler? 

Any idea?
Thanks

**Comment by tilln** [comment]
Any idea how to read [documentation](https://github.com/tilln/jmeter-iso8583?tab=readme-ov-file#sampler)?

> The sampler needs an [ISO8583 Connection Configuration](https://github.com/tilln/jmeter-iso8583?tab=readme-ov-file#config) in its Scope. If there are multiple configuration elements, the optional Connection Reference/Identifier can be used to associate the sampler to a specific one (since v1.2).

> is it possible to add two ISO8583 Connection Configuration"

Yes, then give them individual "Connection Reference/Identifiers".

> how set which packager related to which sampler?

Set the Sampler's "Connection Reference/Identifier" to match the Config element with the correct packager.

**Reply by mehrdad2000** [reply]
@tilln Thanks, miss that part :) work perfectly.
